architecture/design/ysql-tablet-colocation.md

```diff
@@ -1,155 +1,185 @@
 # YSQL Tablet Colocation
 
-> **Note:** This is new feature that is still in design phase.
+> **Note:** This is a new feature that is still in a design phase.
 
 ## Motivation
-A lot of applications using relational databases have a high number of tables and indexes (1000+). These tables are typically small and single database size is less than 500 GB. These applications still want to be able to use distributed SQL database like YugabyteDB to leverage high availability and data resilience.
+
+A lot of applications using relational databases have a high number of tables and indexes (1000+). These tables are typically small, and single database size is less than 500 GB. These applications still want to be able to use a distributed SQL database like YugabyteDB to leverage high availability and data resilience.
```

Comment:
Replace `leverage high availability and data resilience` with `leverage high availability, data resilience and geographic data distribution`